spec/lib/auth/default_current_user_provider_spec.rb

```diff
@@ -750,6 +755,7 @@ def get_cookie_info(cookie_jar, name)
       user_provider = TestProvider.new(env)
       user_provider.log_off_user({}, user_provider.cookie_jar)
       expect(UserAuthToken.find_by(user_id: user.id)).to be_nil
+      expect(event_triggered_user).to eq(user.id)
     end
```

Comment:
I know this can be a debate, but ideally I would create a separate test for this so we can have a good name for the test.

```ruby
context "when logging out the user" do
  it "triggers a user_logged_out event" do
     ...
   end
end
```

Reply:
It makes sense, I‘ve done it.